backend/app/api/endpoints/skill_identity.py

```
@@ -0,0 +1,39 @@
+
+class SkillIdentityVerifyRequest(BaseModel):
+    """Request schema for skill identity verification."""
+
+    token: str = Field(description="Skill identity JWT")
+    user_name: str = Field(description="Claimed user name")
+
+
+@router.post("/verify")
+def verify_skill_identity(
+    request: SkillIdentityVerifyRequest,
+    _: security.AuthContext = Depends(security.get_auth_context),
```

Ruff (0.15.7)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)